tests/integration_tests/tests/utils.py

```diff
@@ -296,3 +296,34 @@ def wait_for_executions(cls):
         attribute = _wait_for_executions_after_call(attribute)
         setattr(cls, name, attribute)
     return cls
+
+
+def get_executions(client, **kwargs):
+    for execution in client.executions.list(_get_all_results=True):
+        if _dict_matches(execution, **kwargs):
+            yield execution
```

Comment:
tbh i'd prefer to avoid having a `_dict_matches` like that:
- `executions.list()` i'm pretty sure already would accept a `workflow_id='..'` filter?
- the parameters check would just be a simple `if` in the loop
- the events kwargs i think also can be just filters to `client.events.list()` already?

Reply:
I've updated both `get_e*` functions.  As for the `_dict_matches` – I wanted to avoid `if`s in the test function body.  I guess it's just an aesthetic preference, but I kind of like it this way.